pkg/features/kube_features.go

```diff
@@ -169,49 +171,61 @@ const (
-	Accelerators:                                {Default: false, PreRelease: utilfeature.Alpha},
-	DevicePlugins:                               {Default: false, PreRelease: utilfeature.Alpha},
-	TaintBasedEvictions:                         {Default: false, PreRelease: utilfeature.Alpha},
-	RotateKubeletServerCertificate:              {Default: false, PreRelease: utilfeature.Alpha},
-	RotateKubeletClientCertificate:              {Default: true, PreRelease: utilfeature.Beta},
-	PersistentLocalVolumes:                      {Default: false, PreRelease: utilfeature.Alpha},
-	LocalStorageCapacityIsolation:               {Default: false, PreRelease: utilfeature.Alpha},
-	HugePages:                                   {Default: false, PreRelease: utilfeature.Alpha},
-	DebugContainers:                             {Default: false, PreRelease: utilfeature.Alpha},
-	PodPriority:                                 {Default: false, PreRelease: utilfeature.Alpha},
-	EnableEquivalenceClassCache:                 {Default: false, PreRelease: utilfeature.Alpha},
-	TaintNodesByCondition:                       {Default: false, PreRelease: utilfeature.Alpha},
-	MountPropagation:                            {Default: false, PreRelease: utilfeature.Alpha},
-	ExpandPersistentVolumes:                     {Default: false, PreRelease: utilfeature.Alpha},
-	CPUManager:                                  {Default: false, PreRelease: utilfeature.Alpha},
-	ServiceNodeExclusion:                        {Default: false, PreRelease: utilfeature.Alpha},
+	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA, Components: []string{KubeProxy, KubeAPIServer}},
+	AppArmor:                                    {Default: true, PreRelease: utilfeature.Beta, Components: []string{Kubelet, KubeAPIServer}},
+	DynamicKubeletConfig:                        {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet, KubeAPIServer}},
+	KubeletConfigFile:                           {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet}},
+	ExperimentalHostUserNamespaceDefaultingGate: {Default: false, PreRelease: utilfeature.Beta, Components: []string{Kubelet}},
+	ExperimentalCriticalPodAnnotation:           {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet, KubeControllerManager}},
+	Accelerators:                                {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet}},
+	DevicePlugins:                               {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet}},
+	TaintBasedEvictions:                         {Default: false, PreRelease: utilfeature.Alpha, Components: []string{KubeControllerManager}},
+	RotateKubeletServerCertificate:              {Default: false, PreRelease: utilfeature.Alpha, Components: []string{Kubelet, KubeControllerManager, KubeAPIServer}},
```

Comment:
while we're at it, can we improve the sorting here so that it's either alphabetical or grouped by component?

Reply:
Sorted alphabetically now.